app/domain/service/contracts.go

```diff
@@ -54,4 +54,13 @@ type Contracts interface {
 	WatchBurnEventLogs(opts *bind.WatchOpts, sink chan<- *abi.RouterBurn) (event.Subscription, error)
 	// WatchLockEventLogs creates a subscription for Lock Events emitted in the Bridge contract
 	WatchLockEventLogs(opts *bind.WatchOpts, sink chan<- *abi.RouterLock) (event.Subscription, error)
+	// FeeAmountFor returns fee data for specific bridge operation
+	FeeAmountFor(_targetChain *big.Int, _userAddress common.Address, _tokenAddress common.Address, _amount *big.Int) (*big.Int, error)
+
+	TokenFeeData(_token common.Address) (struct {
+		ServiceFeePercentage *big.Int
+		FeesAccrued          *big.Int
+		PreviousAccrued      *big.Int
+		Accumulator          *big.Int
+	}, error)
```

Comment:
Do we need these params to be underscored for some reason or is coming from somewhere else?

Reply:
It is copy/paste from router contract proxy in `app/clients/evm/contracts/router/diamond-router.go`. It is easier to change the definition here if the proxy is changed.
I will add the comment above `TokenFeeData `

Reply:
Yeah I see that there they are underscored but here all of the other functions are without underscores so I think it will be best to keep them without them here.